src/arch/mips/include/asm/hal/arch.h

```diff
@@ -3,7 +3,7 @@
 #endif /* HAL_ARCH_H_ */
 
 #ifndef __BYTE_ORDER__
-#define __BYTE_ORDER __BIG_ENDIAN
+#define __BYTE_ORDER __LITTLE_ENDIAN
```

Comment:
It's required smarter definition

Reply:
I dont understand why, but it works even if I define byte order as big endian, so I'll change it back